slices/bash.yaml

```
@@ -0,0 +1,21 @@
+package: bash
+
+slices:
+  config:
+    contents:
+      /etc/bash.bashrc:
+      /etc/skel/.bash_logout:
+      /etc/skel/.bashrc:
+      /etc/skel/.profile:
+
+  bins:
+    essential:
+      - bash_config
+      - libc6_libs
+      - libtinfo6_libs
+    contents:
+      # user should manually link /bin/sh to /bin/bash
+      /bin/bash:
+      /bin/bashbug:
```

```suggestion
      /usr/bin/bashbug:
```

Actually there's a typo here

---

I found the `bashbug` is in two places, I will just create a symlink for this, thanks!

```
$ ls -lah /bin/bashbug
-rwxr-xr-x 1 root root 6.7K Jan  6  2022 /bin/bashbug
$ ls -lah /usr/bin/bashbug
-rwxr-xr-x 1 root root 6.7K Jan  6  2022 /usr/bin/bashbug
```

---

The symlink was rightfully ditched so we can resolve this anyway ;)